src/devices/SoftPwm/SoftwarePwmChannel.cs

```diff
@@ -74,9 +75,9 @@ public override double DutyCyclePercentage
         /// <param name="pinNumber">The GPIO pin number to be used</param>
         /// <param name="frequency">The frequency in hertz. Defaults to 400</param>
         /// <param name="dutyCyclePercentage">The duty cycle percentage represented as a value between 0.0 and 1.0</param>
-        /// <param name="precisionTimer"><see langword="true"/> to use a precision timer. <see langword="false"/> otherwise</param>
+        /// <param name="usePrecisionTimer">Determines if a high precision timer is used.</param>
```

Comment:
small NIT:
`Determines if a high precision timer is used.` -> `Determines if a high precision timer should be used.`?